### DISCUSSION TITLE
GPUProfiler data aggrigation and analysis

### DISCUSSION THREAD

**Discussion by DaddyBurrito** [JeremyMain/GPUProfiler#165] [Q&A]
First off, I would like to thank you Jeremy for the good work you have done with GPU profiler. I wanted to reach out to the community and see how folks are handling the data. I have collected GPD files from 40 workstations and have converted the GPD files to CSV and compiled them into Excel. Now I am trying to develop an appropriate analysis of the data to help guide our team as we start building out our own virtualization platform. The challenge I am encountering is how to translate the large trove of data into actionable decision-driving data. I am interested to learn how you and others have approached this?

**Comment by JeremyMain** [comment]
@DaddyBurrito 
For my own purposes it has been to understand what are the average and peak utilization demands as well as allowing classification of user types by resource requirements.

**Comment by stitchpc** [comment]
Hi @DaddyBurrito.

It really depends on what you want to analyze and why but...
I've done the same, ie: export/CSV/analysis, etc and after considering many options I ended up creating a Normal Curve,. What is important to do is remove the outliers otherwise average calculation will steer analysis is the wrong direction. For example, if one of PC  component is idle for too long, you will get a very low average number and peak numbers will not be easily visible. If you create a time chart, you will end up the same as GPU Profiler does already very well. What you need to do is view the trends and analyze at high level otherwise you will end up wil lots of variables that shoots in all directions. I'm happy to entertain the conversation further if you want to chat...
